operations/mimir-mixin/dashboards/compactor.libsonnet

```diff
@@ -1,6 +1,128 @@
 local utils = import 'mixin-utils/utils.libsonnet';
 
+// Panel query override functions
+local overrideFieldByName(fieldName, overrideProperties) = {
```

Comment:
Could you move all these generic helpers to `operations/mimir-mixin/dashboards/dashboard-utils.libsonnet`? Then you can just reference them via, for example, `$.overrideFieldByName()`.